### PR TITLE
Terminology

### PR DESCRIPTION
**Pull request by adambutler**
Adds a reference terminology page that covers all exposed models, objects, paradigms & interfaces.

### CODE REVIEW

**Comment by adambutler** [comment]
@nicholedwight Really great copywriting, I've made a couple of adjustments please :eyes: 

@nouveller Can you see if we've missed anything

**Comment by nicholedwight** [comment]
🐙

File: getting-started/terminology.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: Terminology
+---
+
+# Terminology
+
+Before you dive into things, you might want to have a look at our terminology guide. Since Rusic is still undergoing changes, the terminology has developed with it.
+
+### Buckets and Spaces
+
+Throughout the docs you'll come across `buckets` quite a lot this is just another word for a `space`, it's your little nook in the Rusic web. So when you're developing your theme or project, you'll be using the term `space` instead, but when working with the API here in the docs, you'll be using `bucket`. A `space` can essentially be thought of as a site, it's your own spot on the web.
```
Comment (nouveller) [inline getting-started/terminology.md]: :fist:

**Comment by nouveller** [comment]
:+1:

**Pull request merged**